packages/framework/package.json

```diff
@@ -24,13 +35,15 @@
     "lint:fix": "pnpm lint -- --fix",
     "format": "prettier --check --ignore-path .gitignore .",
     "format:fix": "prettier --write --ignore-path .gitignore .",
-    "build": "tsup && pnpm check:circulars",
+    "build": "NODE_ENV=production tsup",
     "build:watch": "tsup --watch",
+    "postbuild": "./scripts/copy-package-json.sh && pnpm run check:exports && pnpm check:circulars",
+    "check:exports": "attw --pack .",
+    "check:circulars": "madge --circular --extensions ts --exclude ../../shared ./src",
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Could we remove the `exclude` statement now? The command will error our for bad shared circulars, but I don't think it's such a bad thing to catch it here too.‏
```suggestion
    "check:circulars": "madge --circular --extensions ts ./src",
```

Reply:
Yes, I am planning to do that in the circular dependencies PR. Right now, they are independent and the plan is for this to be merged first.

Reply:
Perfect